### ISSUE TITLE
Catch cellphone cases that act as mirrors.

### ISSUE THREAD

**Issue by lgarron** [thewca/wca-regulations#754]
From the Hefei Summer 2019 report:

> During the comp, I have discovered that there has been an uprising usage of cellphone-recording competitors. Even though most smartphones do not reflect, many smartphones with a clear plastic protecting case can act as a reflecting mirror. 

This is definitely not allowed, but I can see it happening by accident. I think we should consider finding a way to encourage making judges aware of this.

**Comment by xsrvmy** [comment]
where do we draw the line? Solid cases and even the back of a camera screen can be reflective.

**Comment by CarterKoala** [comment]
Sometimes I think about this reg as a whole and think about its usefulness. In theory it makes sense, the front facing cameras/mirrors/reflective phone cases? make more info available to the competitor. 

However, that very same info can be found with a slight tilt of the cube. Which takes more time - tilting the cube slightly or looking completely away from the cube to look in a mirror to see where things are (not to mention figuring out how the mirror affects the pieces - I've tried mirror solves before and it isn't exactly easy) and looking back to the cube to figure out where you are?

Basically what I'm saying in this huge wall of text is that there is no advantage gained from it.

I think that it's fine. Turning a screen around is an easy fix - a reflective phone is not.

There is maybe something to be said about taking out this reg, as it seems to just cause unnecessary DNFs for unfortunate souls who forget, but that's not what this thread is for so I'll leave it at that.

**Comment by xsrvmy** [comment]
@CarterKoala I am actuly in support of simply ignoring this issue for now. Given that both Moyu and Qiyi now require video for prize money it's not surprising that more people are recording in case of an Chinese NR. A lot of phones are reflective without a case these days, so preventing a competitor from using their recording equipment can mean a loss of several hundred dollars of prize money.

**Comment by Samuel-Baird** [comment]
I think just adding a clause that allows delegates to disallow anything they deem too reflective to be disallowed from being near the solving stations is fine pretty much. I wouldn’t worry about a phone case unless it’s a mirror or something

**Comment by xsrvmy** [comment]
@UnderwaterCuber need to be careful cuz cellphone on table can be deemed to be too reflective, but can't give an advantage because the way it reflects does not make the cube visible in it usually.
